gateway/requests/requests.go

```diff
@@ -19,6 +19,7 @@ type DeleteFunctionRequest struct {
 
 // Secret for underlying orchestrator
 type Secret struct {
-	Name  string `json:"name"`
-	Value string `json:"value,omitempty"`
+	Name      string `json:"name"`
```

Comment:
Why are we using the Secret struct from faas, if we have it in faas-provider too? Can we delete this copy?

Reply:
Yeah, took a look here and it's not being used anywhere. Removing it.